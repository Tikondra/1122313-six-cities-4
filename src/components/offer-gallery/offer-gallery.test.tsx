import * as React from "react";
import * as renderer from "react-test-renderer";
import OfferGallery from "./offer-gallery";

it(`should render OfferGallery`, function () {
  const tree = renderer
    .create(
        <OfferGallery
          pictures={[`img/apartment-02.jpg`, `img/apartment-01.jpg`, `img/apartment-03.jpg`]}
          title={`Beautiful & luxurious apartment at great location`}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
