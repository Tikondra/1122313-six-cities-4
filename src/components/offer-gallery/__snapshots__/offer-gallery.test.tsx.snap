// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render OfferGallery 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Beautiful & luxurious apartment at great location"
        className="property__image"
        src="img/apartment-02.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Beautiful & luxurious apartment at great location"
        className="property__image"
        src="img/apartment-01.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Beautiful & luxurious apartment at great location"
        className="property__image"
        src="img/apartment-03.jpg"
      />
    </div>
  </div>
</div>
`;
